app/src/main/java/no/usn/mob3000/ui/components/auth/PasswordResetContent.kt

```diff
@@ -86,6 +87,7 @@ fun PasswordResetContent(
                     onValueChange = { password = it },
                     label = { Text(stringResource(R.string.auth_reset_password_input)) },
                     placeholder = { Text(stringResource(R.string.auth_login_password_placeholder)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
Color was not applied here before, are you _certain_ this is a necessary change? The theme is mostly just applying the static colors through a color scheme. It's practically a 1 to 1 relationship. You add the static colors to the light color scheme, and you replace the direct calls for the colors to the equivalent.

Reply:
these are nessesery because in lightmode, the text becomes invisible, so these are implimentet to be seen inside the textfield